src/Automation/OutputFileHelper.cs

```diff
@@ -100,7 +99,7 @@ private string GenerateScanId()
         {
             var now = _dateTime.Now;
 
-            var nowString = Invariant($"{now:yy-MM-dd_HH-mm-ss.fffffff}");
+            var nowString = $"{now:yy-MM-dd_HH-mm-ss.fffffff}";
```

Comment:
This string should not be localized since it's formatting in a modified ISO format

Reply:
We can keep `Invariant` or we can use `String.Format`. TSA gets unhappy if we use string interpolation without using `Invariant`. The root issue is that string intrerpolation returns a [FormattableString](https://learn.microsoft.com/en-us/dotnet/api/system.formattablestring?view=netstandard-2.0), which then gets converted to a `string` via an implicit call to `ToString`. This `ToString` call doesn't specify a Culture, so it's flagged as a loc problem. The `Invariant` method (see https://referencesource.microsoft.com/#mscorlib/system/FormattableString.cs,67) is literally just a wrapper that calls `ToString` with `CultureInfo.InvariantCulture` after a null check.

I guess my question might be more along the lines of whether this should be hardcoded in the code or moved to a non-localized resource string

Reply:
As it looks like this is only used as a file name, my instinct would be not to localize, so I'll add this instance of Invariant back.

Reply:
@DaveTryon I've updated this PR such that the instances of string interpolation that definitely don't need to be localized are left untouched, but any instances I wasn't sure about are removed and moved into resx files if necessary.